CHANGES.md

```diff
@@ -32,6 +32,10 @@
 - Add `distance` method to `Vector`
 - [Breaking] Use constants instead of functions for `Vector`s' "presets"
 - Add an optional method to `State` to handle error logging
+- [Breaking] Remove `Line::new_sized`
+- [Breaking] Remove all the `newv` functions
```

Comment:
This line should probably include the clarification from your PR; that the `new` functions have been replaced by the `newv` functions

Reply:
I will include that